content/docs/ops/runbook/troubleshooting-kubernetes.md

```diff
@@ -14,26 +14,39 @@ Alerts are generated whenever a pod's status is not `Running`. Alerts contains t
 
 If the namespace is `default` and the pod name is a random string then it is a managed service and likely only impacts a specific tenant application.  If the namespace is `kube-system` or the pod name is human readable then it is a platform component which could impact all managed services.
 
-#### Fixing non-running pods
-Login to a kubernetes master and describe the pod:
+#### Login to kubernetes master
+
+From a jumpbox in the appropriate environment:
+- ssh to a master
+- update the PATH to find the kubernetes binaries
+- sudo to root for all the `kubectl` commands
+
 ```sh
-/var/vcap/packages/kubernetes/bin/kubectl --namespace :namespace describe pod :pod-name
+bosh -d kubernetes ssh master/0
+PATH=$PATH:/var/vcap/packages/kubernetes/bin
+sudo bash
```

Comment:
Shouldn't these two commands be reversed because the sudo command will run in a new shell with the updated variable? Also we need to `export` it as well right?

```sh
$ sudo bash # this is the same as `sudo -i` right?
$ export PATH=$PATH:/var/vcap/packages/kubernetes/bin
```